### PR TITLE
i.MX8MP A53 w/o jailhouse

### PR DESCRIPTION
**Pull request by MrVan**
This PR is to support i.MX8MP AARCH64 EVK.

### CODE REVIEW

**Review by carlocaione — DISMISSED** [review]

File: boards/arm64/imx8mp_evk/imx8mp_evk.dts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+#include <nxp/nxp_imx8mp_a53.dtsi>
+
+/ {
+	model = "NXP i.MX8MP A53";
+	compatible = "fsl,imx8mp";
+
+	aliases {
+		uart-2 = &uart2;
+		uart-4 = &uart4;
+	};
```
Comment (MaureenHelm) [inline boards/arm64/imx8mp_evk/imx8mp_evk.dts:18]: we've generally moved away from using aliases like this. do you still need these?

File: boards/arm64/imx8mp_evk/imx8mp_evk_jailhouse.dts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+#include <nxp/nxp_imx8mp_a53.dtsi>
+
+/ {
+	model = "NXP i.MX8MP A53";
+	compatible = "fsl,imx8mp";
+
+	aliases {
+		uart-2 = &uart2;
+		uart-4 = &uart4;
+	};
```
Comment (MaureenHelm) [inline boards/arm64/imx8mp_evk/imx8mp_evk_jailhouse.dts:18]: we've generally moved away from using aliases like this. do you still need these?

File: boards/arm64/imx8mp_evk/imx8mp_evk.dts (hunk 1 above)
Comment (MrVan) [reply boards/arm64/imx8mp_evk/imx8mp_evk.dts]: Dropped.

File: boards/arm64/imx8mp_evk/imx8mp_evk_jailhouse.dts (hunk 2 above)
Comment (MrVan) [reply boards/arm64/imx8mp_evk/imx8mp_evk_jailhouse.dts]: Dropped.

**Comment by MrVan** [comment]
@carlocaione Your approval was deleted, after I push a minor change to address @MaureenHelm's comments. Not sure whether it is github behavior, please give a look again of this PR. Thanks!

**Review by carlocaione — APPROVED** [review]

**Review by povergoing — APPROVED** [review]

**Pull request merged**